<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUI</class>
 <widget class="QWidget" name="SettingsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>775</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QGroupBox" name="chartsBox">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Database</string>
     </property>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>741</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Database Options</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="userpath">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>481</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="shpLabel_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordpath">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>481</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="shpLabel_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>131</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="portpath">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>481</width>
         <height>35</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="shpLabel_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>91</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="hostpath">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>481</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="shpLabel_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>101</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Host</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Driver</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="driverCB">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>MySQL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PostgreSQL</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="geometry">
        <rect>
         <x>514</x>
         <y>242</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>741</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Chart</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect to database</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>DataBase</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Editor">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>570</y>
        <width>731</width>
        <height>181</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Chart settings file</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QPushButton" name="pushButton_chartTableEditor">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Open chart file editor</string>
       </property>
       <property name="text">
        <string>Editor &gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_chartTableXMLFile">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>561</width>
         <height>35</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_browseChartTableXMLFile">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>70</y>
         <width>61</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/fileopen.png</normaloff>:/fileopen.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>File name (XML)</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
